app/assets/stylesheets/style_guide/modules/_hero_simple.scss

```diff
@@ -0,0 +1,11 @@
+.hero_simple-sample {
+  margin-top: $header-height;
+  background-image: image-url("style_guide/hero/beach-girl.jpg");
+  background-size: contain;
+  background-position: 0% -15%;
+
+  @media (min-width: $screen-sm) {
```

Comment:
I think it would be valuable to include some additional breakpoints here at which we would serve optimized images ex. beach-girl.jpg, beach-girl-md.jpg, beach-girl-lg.jpg so that the sample markup suggests we should always be providing optimized images and its' more efficient to do so. Does that seem reasonable to you?